bin/tessel-2.js

```diff
@@ -24,6 +24,7 @@ function makeCommand(commandName) {
     .option('key', {
       required: false,
       metavar: 'PRIVATEKEY',
+      default: controller.LOCAL_AUTH_KEY,
```

Comment:
Can you add `var Tessel = require('../lib/tessel/tessel');` to this file and use `Tessel.LOCAL_AUTH_KEY` directly?